src/System.Net.Security/tests/FunctionalTests/NegotiateStreamTestForUnix.cs

```diff
@@ -46,7 +47,7 @@ public KDCSetup()
                         Dispose();
                     }
 
-                    Assert.True(false, "KDC setup failure");
+                    _successfulSetup = false; // TODO: https://github.com/dotnet/corefx/issues/12107
```

Comment:
Do we still need this TODO? Moreover it is closed, so we should remove it also from 2.0, right? Or did I misunderstand?